benchmarking.py

```diff
@@ -0,0 +1,56 @@
+import matplotlib.pyplot as plt
+import timeit
+import csv
+
+class Graph:
+    #static 2d array to store data
+    taskData = [[],[],[],[],[]]
+
+    #initialize function to start our graph
+    def __init__(self):
+        self.title = 'Tasks per Request vs Average Reward'
+
+    #add data point for the graph. Use x to specify the tasks per request
+    def addPoint(self, x, y):
+        taskData[x].append(y)
+
+    #plot graph when finished, export to benchData
+    def plot(self, numTasks):
+        xvals = []
+        yvals = []
+        for x in range(0, numTasks):
+            xvals.append(x)
+            yvals.append(sum(taskData(x))/len(taskData(x)))
+        plt.plot(xvals,yvals)
+        plt.xlabel('Tasks per Request')
+        plt.ylabel('Average Reward')
+        plt.title(self.title)
+        plt.savefig('benchData/taskVreward.png')
```

Comment:
Do we have any idea what this png looks like currently? Can you add that to the PR? @kylevitale2

Reply:
I haven't tested this yet, I was unsure of where to pull the rewards from, and during the times I ran server.py and client.py it seemed I only got one run, which can be graphed but defeats the average aspect. I'll do tests on this soon.